packages/suite-desktop/src-electron/bridge.ts

```diff
@@ -12,46 +12,23 @@ const error = (msg: string | Error) => {
     throw new Error(`cannot run bridge binary - ${msg}`);
 };
 
-const getArch = () => {
+const getSystemExt = () => {
+    let platform = os.platform();
+    if (platform === 'darwin') platform = 'mac';
+    if (platform === 'win32') platform = 'win';
     const arch = os.arch();
-    switch (arch) {
-        case 'x32':
-            return 'x32';
-        case 'x64':
-            return 'x64';
-        default:
-            error('unsupported system architecture');
+    const ext = platform === 'win' ? '.exe' : '';
+    const system = `${platform}-${arch}`;
+    const supportedSystems = ['linux-x32', 'linux-x64', 'mac-x64', 'win-x32', 'win-x64'];
+    if (supportedSystems.includes(system)) {
+        return [system, ext];
     }
+    error('unsupported system');
```

Comment:
maybe it would be nice to know detected unsupported system? (for debugging/support purposes) 
```
error(`unsupported system ${platform} ${arch}`)
```

Reply:
Addressed in d33dcded8ddfea82ab19564f5fa027606bb4e15d